### ISSUE TITLE
Switching Corral to use lit for running regressions

### ISSUE THREAD

**Issue by zvonimir** [boogie-org/corral#121]
It would probably be a good idea to switch to using LLVM's lit tool to run regressions (https://llvm.org/docs/CommandGuide/lit.html), the same way Boogie does.

I am kind of familiar with it now, and so I can make this switch if others are ok with it.
We would add a dependency on lit, but lit is super easy to install using Python's pip package manager.

**Comment by zvonimir** [comment]
@akashlal Any thoughts on this update?

**Comment by akashlal** [comment]
Makes sense to me to use the same tool as Boogie. It would be a great improvement over existing regressions.
